docs/src/main/asciidoc/kafka.adoc

```diff
@@ -566,6 +569,53 @@ mp.messaging.incoming.your-channel.group.id=${quarkus.uuid}
 
 IMPORTANT: If the `group.id` attribute is not set, it defaults the `quarkus.application.name` configuration property.
 
+=== Receiving Kafka Records in Batches
+
+By default, incoming methods receive each Kafka record individually.
+Under the hood, Kafka consumer clients poll the broker constantly and receive records in batches, presented inside the `ConsumerRecords` container.
+
+In *batch* mode, your application can receive all the records returned by the consumer *poll* in one go.
+
+To achieve this you need to specify a compatible container type to receive all the data:
+
+[source, java]
+----
+@Incoming("prices")
+public void consume(List<Double> prices) {
+    for (double price : prices) {
+        // process price
+    }
+}
+----
+
+The incoming method can also receive `Message<List<Payload>>`, `KafkaBatchRecord<Key, Payload>` `ConsumerRecords<Key, Payload>` types.
```

Comment:
This sentence is still different from the following example: `KafkaBatchRecord` vs. `KafkaRecordBatch`.

Reply:
I guess I couldn't wrap my head around the name change :)